### DISCUSSION TITLE
Help in deploying serverless

### DISCUSSION THREAD

**Discussion by siddharth223437** [gaiin-platform/.github#5] [General]
Hello, i am trying to deploy this to aws but i am continuously getting below errors while deploying amplify-genai-backend.

1. No export named dev-RestApiId found
2. No export named dev-RestApiRootResourceId found 
3. Cannot resolve variable at "provider.environment.API_BASE_URL": Value not found at "self" source

Please help me in resolving this errors.

Thanks,
Sid

**Comment by karnsab** [comment]
@siddharth223437 and I connected via email! they are up and running.
